### DISCUSSION TITLE
Any plans for next release for v6 patch

### DISCUSSION THREAD

**Discussion by grinish21** [golangci/golangci-lint-action#1111] [Q&A] [answered]
are there plans to release a new patch soon with the dependabot version bumps? the last release was in jul 29.

**Comment by ldez** [comment]
There is nothing that really required a release inside the dependabot updates, why are you asking that?

**Reply by grinish21** [reply]
I am behind a corporate proxy and while pulling the latest tag there are dependabot scan warnings and i cannot pull the latest. most of the warnings are merged but not in the latest release.

**Reply by ldez** [reply, marked as answer]
I created a new release.

I hope you enjoyed our work, please consider donating or asking your company to do so.
This will be appreciated, thank you :heart:

[![Open Collective backers and sponsors](https://img.shields.io/badge/OpenCollective-Donate-blue?logo=opencollective&style=for-the-badge)](https://opencollective.com/golangci-lint)
[![GitHub Sponsors](https://img.shields.io/badge/GitHub-Donate-blue?logo=github&style=for-the-badge)](https://github.com/sponsors/golangci)
